js/src/components/tree-select-control/index.scss

```diff
@@ -161,7 +165,17 @@ $muriel-box-shadow-8dp: 0 5px 5px -3px rgba(0, 0, 0, 0.2),
 
 		&.is-selected,
 		&:hover {
-			background: $studio-gray-0;
+			color: $studio-wordpress-blue-50;
+		}
+	}
+
+	.woocommerce-tree-select-control__expander-icon {
+		cursor: pointer;
+		margin-right: 0;
+
+		&.is-hidden {
+			visibility: hidden;
+			pointer-events: none;
 		}
```

Comment:
For nodes without children, we show still the space of the Expand Icon for alignment.

Reply:
How about adding this as code comment as well?